tests/integration/test_pagure.py

```diff
@@ -175,6 +175,15 @@ def test_get_web_url(self):
 
     def test_full_repo_name(self):
         assert self.ogr_project.full_repo_name == "ogr-tests"
+        assert (
+            self.service.get_project(namespace="mbi", repo="ansible").full_repo_name
+            == "mbi/ansible"
+        )
+
+        # test fork
+        assert self.ogr_fork.exists()
+        assert self.ogr_fork.is_fork
```

Comment:
I think that is_fork probably doesn't have to be there

Reply:
It should be `if exists then assert` for fork, right? If I don't have fork, I can't query and compare the result.
edit: can be removed, since it's checking against the saved responses

Reply:
> edit: can be removed, since it's checking against the saved responses

:+1: (I was not sure -- you did not need to change that..;)